corehq/apps/accounting/models.py

```diff
@@ -2711,6 +2711,13 @@ def get_credits_by_subscription_and_features(cls, subscription,
         ).all()
 
     @classmethod
+    def get_non_general_credits_by_subscription(cls, subscription):
```

Comment:
@nickpell I have updated non-general credit query per your comment. Does this look right now?